demo/img_demo.py

```diff
@@ -74,6 +74,8 @@ def main():
         if args.out_img_root == '':
             out_file = None
         else:
+            if not os.path.isdir(args.out_img_root):
```

Comment:
check this https://docs.python.org/3/library/os.html#os.makedirs

Reply:
how about setting `exist_ok=True`

Reply:
OK.